03-attribute-operations.Rmd

```diff
@@ -232,25 +232,31 @@ Another advantage over the nesting approach is that you can easily comment out c
 **dplyr** works especially well with the pipe operator because its fundamental functions (or 'verbs', like `select()`) expect a data frame object as input and also return one.^[If you want **dplyr** to return a vector, use `pull`.]
 
 ### Vector attribute aggregation
+Aggregation operations summarize datasets in accordance with a grouping variable.
+Lets illustrate this with an example.
+We would like to calculate the number of people per continent. 
+Fortunately, our `world` dataset has one column representing the inhabitants per country and one column representing the corresponding continent. 
+Hence, we can take the sum of the populations per country using the continent column as a grouping variable. 
+In base R the `aggregate()` function lets you do that.
+Note that you have to indicate the grouping variable as a `list`-object:
 
-<!-- https://github.com/ropenscilabs/skimr ?? -->
-
-<!-- As demonstrated in chapter \@ref(spatial-class), `summary()` provides a quick summary of the spatial and non-spatial components of spatial objects.
-Enter the following command for an overview of the `world` object and all its variables (result not shown):
-
-```{r, eval=FALSE}
-summary(world)
+```{r, eval = FALSE}
```

Comment:
I have swapped places, i.e. I start with a base R example followed by a tidyverse example, just to remain consistent.

Reply:
Great plan, looks good.